db/sql/00_msar.sql

```diff
@@ -310,13 +241,13 @@ Args:
   rel_id:  The OID of the relation.
   col_name:  The unquoted name of the column in the relation.
 */
-SELECT quote_ident(attname::text) FROM pg_attribute WHERE attrelid=rel_id AND attname=col_name;
+SELECT attname::text FROM pg_attribute WHERE attrelid=rel_id AND attname=col_name;
```

Comment:
We'd probably also need a refactor for qualifying the catalog table with `pg_catalog`

Reply:
Yeah, I see that as unrelated to this PR though.

Reply:
Yep, definitely not worth pursuing in this PR.